Tests/Functional/Drivers/Queue/StompQueueDriverTest.php

```diff
@@ -20,6 +19,7 @@ class StompQueueDriverTest extends AbstractQueueDriverTest
      * @dataProvider getMessages
      *
      * @param MessageInterface $msg
+     * @group stomp1
```

Comment:
There is a typo in the `@group` (the name of the group is `stomp1` and I guess you wanted to define the name as `stomp`)

Reply:
Fixed.